great_expectations/datasource/fluent/sqlite_datasource.py

```diff
@@ -109,6 +110,13 @@ def __init__(self, **kwargs):
 
     type: Literal["table"] = "table"
 
+    @override
+    def validate_batch_definition(self, partitioner: ColumnPartitioner) -> None:
```

Comment:
These sqlite assets should be public but aren't. I will mark them public and put a public api marker on these methods in a different PR.